article/app/model/KeyEventData.scala

```diff
@@ -7,21 +7,20 @@ import com.github.nscala_time.time.Imports._
 object KeyEventData {
 
   // just for convenience for use from the templates
-  def apply(maybeBlocks: Option[Blocks], timezone: DateTimeZone): Seq[KeyEventData] = {
+  def apply(maybeBlocks: Option[Blocks], timezone: DateTimeZone, filterKeyEvents: Boolean): Seq[KeyEventData] = {
 
     val blocks =
       maybeBlocks.toSeq.flatMap(blocks => blocks.requestedBodyBlocks.getOrElse(CanonicalLiveBlog.timeline, blocks.body))
 
-    apply(blocks, timezone)
+    apply(blocks, timezone, filterKeyEvents)
   }
 
-  def apply(blocks: Seq[BodyBlock], timezone: DateTimeZone): Seq[KeyEventData] = {
-
+  def apply(blocks: Seq[BodyBlock], timezone: DateTimeZone, filterKeyEvents: Boolean): Seq[KeyEventData] = {
     val TimelineMaxEntries = 7
     val latestSummary = blocks.find(_.eventType == SummaryEvent)
     val keyEvents = blocks.filter(_.eventType == KeyEvent)
-    val bodyBlocks =
-      (latestSummary.toSeq ++ keyEvents).sortBy(_.publishedCreatedTimestamp).reverse.take(TimelineMaxEntries)
+    val filteredBlocks = if (filterKeyEvents) keyEvents else latestSummary.toSeq ++ keyEvents
```

Comment:
add a note to PR to explain that there is a separate task to include summary posts in the filtered list

Reply:
@joecowton1

Reply:
Done.